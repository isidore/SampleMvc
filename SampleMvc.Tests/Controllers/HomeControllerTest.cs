using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using SampleMvc.Controllers;
#if DEBUG
namespace SampleMvc.Tests.Controllers
{
	[TestClass]
	[UseReporter(typeof(FileLauncherReporter), typeof(DiffReporter))]
	public class HomeControllerTest : MvcTest
	{
		[TestMethod]
		public void About()
		{
			MvcApprovals.VerifyMvcPage(new HomeController().About);
		}

		[TestMethod]
		public void TestStaticVersionOfStatus()
		{
			MvcApprovals.VerifyMvcPage(new HomeController().TestStaticVersionOfStatus);
		}
	}
}
#endif