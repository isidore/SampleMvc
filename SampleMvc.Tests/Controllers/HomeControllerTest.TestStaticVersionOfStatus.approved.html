

<!DOCTYPE html>

<html>
	<head><base href="http://localhost:50011/Home">
		<title>Status</title>
	</head>
	<body>
		<h1>Report for Customer Lynn</h1>
		<h3>Shipped Orders:</h3>
			<div># A1201</div>
			<div># A1202</div>

		<h3>Partially Shipped Orders:</h3>
			<div># A1203: waiting 
Hiking Boots			</div>
			
			
		<h3>Unshipped Orders:</h3>
			<div># A1205: 2 items</div>
	
	</body>
</html>